src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -476,6 +477,18 @@ const Stream = createReactClass({
         this.transitionTo
       );
     }
+
+    // Ignore saved searches
+    if (this.state.savedSearchList.map(s => s.query == this.state.query).length > 0) {
```

Comment:
How come we need to go through this map? Instead of grabbing `this.state.query` directly?

Reply:
To me, issue search means "user typed in a query and pressed search." Loading a Saved Search (effectively a dashboard) feels less like running a search and more like a pageview (which is why today, it's caught as such).

Reply:
@ehfeng I don't get it, what's the point of map if you just want the length of the array?

Reply:
@lynnagara oh, as opposed to doing...a filter? or match? I could switch it to those as well—I can switch to that if you want.

Reply:
Do you want to check if the query matches any saved search? I'd use `Array.find` for that. It'll always be true right now unless they've gone and deleted every single saved search. Is it important to filter these out in any case? Loading a saved search won't trigger this function anyway.